frontend/pixi.toml

```diff
@@ -0,0 +1,17 @@
+[project]
+name = "frontend"
+version = "0.1.0"
+description = "Add a short description here"
+authors = ["Yousifff <yousifsayer@gmail.com>"]
+channels = ["conda-forge"]
+platforms = ["linux-64"]
+
+[tasks]
+admin = "pixi run streamlit run pages/admin.py"
```

Comment:
You should just create a task `app = "pixi run streamlit run main.py"`

Streamlit will then identify the pages and run the app appropriately.

Reply:
Yes, it's raising error actually this why I added admin and index in pixi file.
I'm trying to solve it.